sse/README.md

```diff
@@ -0,0 +1,63 @@
+# Server Sent Events Bindings
+
+This document defines how to describe Server Sent Events-specific information on AsyncAPI.
+
+See the [Server-Sent Events protocol specification][protocolSpecification].
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+
+<a name="server"></a>
+
+## Server Binding Object
+
+This object MUST NOT contain any properties. Its name is reserved for future use.
+
+
+<a name="channel"></a>
+
+## Channel Binding Object
+
+When using Server Sent Events, the channel represents a single logical message stream flowing from server to client. 
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="operationBindingObjectMethod"></a>`method` | string | The HTTP method to use when establishing the request. Its value MUST be `GET`.
+<a name="operationBindingObjectQuery"></a>`query` | [Schema Object][schemaObject] \| [Reference Object](referenceObject) | A Schema object containing the definitions for each query parameter. This schema MUST be of type `object` and have a `properties` key.
+<a name="operationBindingObjectHeaders"></a>`headers` | [Schema Object][schemaObject] \| [Reference Object](referenceObject) | A Schema object containing the definitions of the HTTP headers to use when establishing the request. This schema MUST be of type `object` and have a `properties` key.
+<a name="operationBindingObjectBindingVersion"></a>`bindingVersion` | string | The version of this binding. If omitted, "latest" MUST be assumed.
+
+This object MUST contain only the properties defined above.
+
+<a name="operation"></a>
+
+## Operation Binding Object
+
+This object MUST NOT contain any properties. Its name is reserved for future use.
+
+
+<a name="message"></a>
+
+## Message Binding Object
+
+This object contains information about the message representation in SSE.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="messageBindingObjectEventType"></a>`event` | string | Server-sent event type, defaults to `message` if omitted, by the [SSE specification][protocolSpecification]. |
```

Comment:
I think this is the only part of this binding that's truly related to SSE. The rest is HTTP. So my proposal is that you add this info to the HTTP message binding instead. We already have `headers` and `statusCode`. We can add `sse` which in turn will contain `eventType`.

Reply:
Agreed this is majority of it, though there is protocol framing in the HTTP response, and the optional `last-event-id` request header is implicitly defined for SSE requests.

Actually, this header represents a cursor indicating progress into the SSE stream, allowing a disconnected client to recover from where they left off. So the logical SSE message stream is potentially spanning multiple HTTP requests.

If not abstracted out by `sse` protocol, how would the definition of acceptable HTTP headers that be handled by the proposed approach to use the HTTP binding for SSE message streams?

Reply:
Fair enough. I've never used SSE myself so I'm just speaking from a theoretical point of view.